packages/core/src/css_composer/view/CssRuleView.ts

```diff
@@ -13,7 +13,7 @@ export default class CssRuleView extends View<CssRule> {
     this.listenTo(model.get('selectors'), 'change', this.render);
   }
 
-  // @ts-ignore
+  /** @ts-ignore */
   tagName() {
     return 'style';
   }
```

Comment:
For some reason build fails if we don't add the comment: 

```
/** @ts-ignore */
```

I see we also have to do this in `ComponentView` file. 

Removing the ignore results in build error: 

```
Property 'tagName' in type 'ComponentView<TComp>' is not assignable to the same property in base type 'View<TComp, HTMLElement>'.
  Type '() => string' is not assignable to type 'string'.ts(2416)
Class 'View<TComp, HTMLElement>' defines instance member property 'tagName', but extended class 'ComponentView<TComp>' defines it as instance member function.
```


```

Property 'tagName' in type 'CssRuleView' is not assignable to the same property in base type 'View<CssRule, HTMLElement>'.
  Type '() => string' is not assignable to type 'string'.ts(2416)
Class 'View<CssRule, HTMLElement>' defines instance member property 'tagName', but extended class 'CssRuleView' defines it as instance member function.ts(2425)
```

Changing the method to a string then results in failing tests.

Reply:
yeah I don't remember if it was an issue on our side or the TS from the backbone, but it's ok, not a blocker